lua/cfcfactions/core/cl/cl_clientstartup.lua

```diff
@@ -44,27 +44,35 @@ function cfcFactions:DisplayMenu()
 end
 
 --If the defined key is properly set, users can use that specific key to also open/close the derma
-hook.Add("PlayerButtonDown", "CFC_FAC_MENUKEYDOWN", function(ply, button)
+local function menuKeyDown(ply, button)
     if input.GetKeyName(button) == cfcFactions.Config.CLIENT_KEY then
         cfcFactions:DisplayMenu()
     end
-end)
+end
+
+hook.Add("PlayerButtonDown", "CFC_FAC_MENUKEYDOWN", menuKeyDown)
 
 --If a user requests to display the derma from serverside
-net.Receive('CFC_Fac_ToggleDerma', function(len, ply)
+local function toogleDerma(len, ply)
     cfcFactions:DisplayMenu()
-end)
+end
+
+net.Receive('CFC_Fac_ToggleDerma', toogleDerma)
 
-net.Receive('CFC_Fac_SendMessage', function(len, ply)
+local function sendMessage(len, ply)
```

Comment:
```suggestion
local function sendFactionsMessage(len, ply)
```

Can you rename the rest of them in this manner, too?